### PR TITLE
Emulation-clip-zero

### PR DESCRIPTION
**Pull request by nbren12**
Removes any emulator updates instead of "cheating" by looking up the true values.

I also added a command line tool `query-wandb.py reproduce` for grabbing ready-to-submit configurations from wandb. Currently it only supports the prognostic run.

Coverage reports (updated automatically):
- test_unit: [83%](https:\/\/output.circle-artifacts.com\/output\/job\/a2326983-3d96-48e8-a0db-420351abf6bd\/artifacts\/0\/tmp\/coverage\/htmlcov-test_unit\/index.html)

### CODE REVIEW

File: external/emulation/emulation/masks.py [hunk 1]
```diff
@@ -40,14 +40,28 @@ def __call__(self, state: FortranState, emulator: FortranState) -> FortranState:
...
+        self.fill_value = fill_value
 
     def __call__(self, state: FortranState, emulator: FortranState) -> FortranState:
         use_fortran_state = slice(self.start, self.stop)
         # Fortran state TOA is index 79, and dims are [z, sample]
         emulator_field = np.copy(emulator[self.key])
-        emulator_field[use_fortran_state] = state[self.key][use_fortran_state]
+        if self.fill_value is None:
+            emulator_field[use_fortran_state] = state[self.key][use_fortran_state]
+        elif isinstance(self.fill_value, str):
```
Comment (frodre) [inline external/emulation/emulation/masks.py:61]: What is the `str` case for?

**Review by frodre — APPROVED** [review]
LGTM!

File: projects/microphysics/scripts/query-wandb.py [hunk 2]
```diff
@@ -19,6 +21,28 @@ def groups(experiment: str, filter_tags: str = typer.Option("", "-f")):
         print(group)
 
 
+def wandb2job(run: Any) -> end_to_end.ToYaml:
+    if run.job_type == "prognostic_run":
+        return end_to_end.PrognosticJob(
+            name=run.name,
```
[the remark this answers is not in the document]
Comment (nbren12) [reply projects/microphysics/scripts/query-wandb.py:27]: The argo workflow name is actually different than this one. Maybe this isn't super useful default, but I envision changing it most of the time.

File: external/emulation/emulation/masks.py (hunk 1 above)
Comment (nbren12) [reply external/emulation/emulation/masks.py:61]: It's how I implemented the "no change" option. `LevelSlice(start=74, end=None, fill_value='air_temperature_input)` will replace the ml output with `air_temperature_input` at the specified levels.

File: external/emulation/emulation/masks.py [hunk 3]
```diff
@@ -40,14 +40,28 @@ def __call__(self, state: FortranState, emulator: FortranState) -> FortranState:
...
         self.stop = stop
+        self.fill_value = fill_value
 
     def __call__(self, state: FortranState, emulator: FortranState) -> FortranState:
         use_fortran_state = slice(self.start, self.stop)
         # Fortran state TOA is index 79, and dims are [z, sample]
         emulator_field = np.copy(emulator[self.key])
-        emulator_field[use_fortran_state] = state[self.key][use_fortran_state]
+        if self.fill_value is None:
+            emulator_field[use_fortran_state] = state[self.key][use_fortran_state]
```
[the remark this answers is not in the document]
Comment (nbren12) [reply external/emulation/emulation/masks.py:60]: I'm not sure exactly which variable you are referring to. I didn't think the code was hard to follow, so I think I'll just leave it the same.
